### PR TITLE
Issue #113 google style

### PR DESCRIPTION
**Pull request by cemsbr**

### CODE REVIEW

File: pyof/v0x01/common/utils.py [hunk 1]
```diff
@@ -90,24 +86,24 @@ def new_message_from_message_type(message_type):
...
 
     Raises:
-        KytosUndefinedMessageType: unkown Message_Type
+        KytosUndefinedMessageType: Unkown Message_Type.
     """
     message_type = header.message_type
     if not isinstance(message_type, Type):
         try:
             if isinstance(message_type, str):
-                message_type = Type[message_type]
```
Comment (cemsbr) [inline pyof/v0x01/common/utils.py]: Bug

File: pyof/v0x01/common/utils.py [hunk 2]
```diff
@@ -36,7 +36,7 @@
 from pyof.v0x01.symmetric.echo_request import EchoRequest
 
 
-__all__ = ['new_message_from_header', 'new_message_from_message_type']
+__all__ = ('new_message_from_header', 'new_message_from_message_type')
```
Comment (beraldoleal) [inline pyof/v0x01/common/utils.py]: Can you explain this, please ? All python docs, including many PEPs, `__all__` is always a list.

File: pyof/v0x01/common/utils.py (hunk 1 above)
Comment (beraldoleal) [inline pyof/v0x01/common/utils.py]: Probably you introduced a bug. message_type here is a string:
```python3
message_type = Type("OFPT_HELLO")
ValueError: 'OFPT_HELLO' is not a valid Type
```

Please check this with @diraol to make sure that this way is right.

Comment (cemsbr) [inline pyof/v0x01/common/utils.py]: You are right. It was a false positive from pylint (https://github.com/PyCQA/pylint/issues/992). I reverted it.

**Pull request merged**